docker/dockerfiles/fe/fe-ubi.Dockerfile

```diff
@@ -26,6 +26,7 @@ FROM artifacts-from-${ARTIFACT_SOURCE} as artifacts
 
 FROM registry.access.redhat.com/ubi8/ubi:8.7
 ARG STARROCKS_ROOT=/opt/starrocks
+ENV STARROCKS_ROOT=${STARROCKS_ROOT}
```

Comment:
FYI This ENV VAR is being used and updated in a few places of the shell scripts.
We need to come to a consensus with @kevincai @yandongxiao  on where is the best places to set them and align the expectation
https://github.com/search?q=repo%3AStarRocks%2Fstarrocks%20STARROCKS_ROOT&type=code

Reply:
Seems like `/opt/starrocks` is the default value in all the files. We might even remove it as an input arg.
> STARROCKS_ROOT=${STARROCKS_ROOT:-"/opt/starrocks"}

Reply:
> We might even remove it as an input arg.

+100 . This will eliminate the ambiguity in the code.

Reply:
STARROCKS_ROOT is only used in build the docker image, all the deployment related path search is using `STARROCKS_HOME`, I'd prefer download the jar into `/release/fe_artifacts/fe/datadog/dd-java-agent.jar`, and use `$STARROCKS_HOME/datadog/dd-java-agent.jar` as the referencing path.

Reply:
The `allin1` image uses [SR_HOME](https://github.com/StarRocks/starrocks/blob/main/docker/dockerfiles/allin1/allin1-ubuntu.Dockerfile#L32) instead, can we put it aside and focus on `be/fe` images only?